src/main/java/com/example/gagansaurabhpractice/controller/ContactController.java

```diff
@@ -0,0 +1,36 @@
+package com.example.gagansaurabhpractice.controller;
+
+import java.util.List;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.gagansaurabhpractice.entity.Contact;
+import com.example.gagansaurabhpractice.service.ContactService;
+
+@RestController
+public class ContactController {
+
+	@Autowired
+	ContactService conService;
+	
+	@GetMapping("/contacts")
+	public List<Contact> getAllContacts(){
```

Comment:
Here, instead of returning `List<Contact>` can we return a specific response model, like we discuss in our doc ? That response model will contains error, status and array of Contact object.